### PR TITLE
Implement modular approach to commands.

### LINKED ISSUE #12: Add support to show knockout fixtures of Champions League as rounds
Explore, if possible, and add support to show knockout fixtures of Champions League as rounds

### PR DESCRIPTION
**Pull request by armansujoyan**
## Description of new feature, or changes

Commands are moved to their corresponding files. This modular approach will make it more clear and easy to use.

## Link to YARGS
[Official documentation of modular approach.](https://github.com/yargs/yargs/blob/master/docs/advanced.md#commands)

## Related Issues and Discussions

Enhancement: Separate commands into modules. #41

## People to notify

@ManrajGrover

### CODE REVIEW

File: cmds/standings.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const request = require('request');
+const URLS = require('../constants');
+const config = require('../config');
+const helpers = require('../helpers');
+const leagueIds = require('../leagueIds');
+
+const updateMessage = helpers.updateMessage;
+const standingsHelper = helpers.standingsHelper;
+
+const footballRequest = request.defaults({
```
Comment (ManrajGrover) [inline cmds/standings.js]: Let's move this to separate utility file for `scores`, `standings` and `fixtures`.

File: cmds/standings.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const ora = require('ora');
+const moment = require('moment');
+const request = require('request');
+const URLS = require('../constants');
+const config = require('../config');
+const helpers = require('../helpers');
+const leagueIds = require('../leagueIds');
+
+const updateMessage = helpers.updateMessage;
+const standingsHelper = helpers.standingsHelper;
```
Comment (ManrajGrover) [inline cmds/standings.js]: `helpers.standingsHelper` -> `helpers.standings`

File: cmds/config.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  }];
+
+  inquirer.prompt(questions).then((answers) => {
+    const obj = config;
+
+    if (answers.API_KEY !== '') {
+      obj.API_KEY = answers.API_KEY;
+    }
+
+    fs.writeFileSync(path.resolve(__dirname, 'config.json'), JSON.stringify(obj, null, 2), 'utf8');
```
Comment (ManrajGrover) [inline cmds/config.js]: This config file should be written in root directory and not in `cmds`. Please fix the same.

File: cmds/standings.js (hunk 1 above)
Comment (armansujoyan) [reply cmds/standings.js]: You mean creating separate files for scores, fixtures and standings instead of having one helper.js right?

Comment (ManrajGrover) [reply cmds/standings.js]: No, let's keep `request_utils.js` which exports this object.

**Pull request merged**